src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java

```diff
@@ -45,10 +49,31 @@ public void testAllelicCapSegger() {
         final File initialSimilarSegmentsFile = new File(outputPrefix + "-" + ACNVModeller.INITIAL_SEG_FILE_TAG + ".seg");
         final File finalSimilarSegmentsFile = new File(outputPrefix + "-" + ACNVModeller.FINAL_SEG_FILE_TAG + ".seg");
 
-        Assert.assertTrue(snpSegmentsFile.isFile());
-        Assert.assertTrue(unionedSegmentsFile.isFile());
-        Assert.assertTrue(noSmallSegmentsFile.isFile());
-        Assert.assertTrue(initialSimilarSegmentsFile.isFile());
-        Assert.assertTrue(finalSimilarSegmentsFile.isFile());
+        final List<File> outputFileList = new ArrayList<>();
+        outputFileList.add(snpSegmentsFile);
+        outputFileList.add(unionedSegmentsFile);
+        outputFileList.add(noSmallSegmentsFile);
+        outputFileList.add(initialSimilarSegmentsFile);
+        outputFileList.add(finalSimilarSegmentsFile);
+
+        for (final File outputFile: outputFileList) {
+
+            // Check that all files are files with a size greater than 0.
+            Assert.assertTrue(outputFile.isFile());
+            Assert.assertTrue(outputFile.length() > 0);
+            try {
+
+                //Check that all files have:
+                //  - at least two lines and all either start with "#" or contain at least one "\t"
+                //  - at least two lines with tab (header + 1 segment)
+                final List<String> outputLines = FileUtils.readLines(outputFile);
+                Assert.assertTrue(outputLines.size() > 2);
+                Assert.assertEquals(outputLines.stream().filter(l -> l.contains("\t") || l.startsWith("#")).count(), outputLines.size());
+                Assert.assertTrue(outputLines.stream().filter(l -> l.split("\t").length > 2).count() > 2, "File: " + outputFile + " does not seem to have at least one segment and a header.");
```

Comment:
probably OK to assume that header lines will not contain two tabs or more, but perhaps add `!l.startsWith("#")` to be safe?